manager/role_manager.go

```diff
@@ -133,12 +133,19 @@ func (rm *roleManager) reconcileRole(node *api.Node) {
 				return
 			}
 
-			rmCtx, rmCancel := context.WithTimeout(rm.ctx, 5*time.Second)
+			rmCtx, rmCancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Does this need to be `context.Background()` if `rm.raft.TransferLeadership` uses a new context?